using-timescaledb/continuous-aggregates.md

```diff
@@ -139,19 +175,98 @@ PostgreSQL. In addition, TimescaleDB continuous aggregates does not
+in the background. You can disable the immediate materialization of
+data using the `WITH NO DATA` option to `CREATE MATERIALIZED VIEW`.
+
+#### Automatic refresh with a continuous aggregate policy
+
+You can refresh the continuous aggregate view manually as mentioned
+above, but you can also automate the refresh by adding a continuous
+aggregate policies to automatically refresh data.
+
+There are a few situations that you might want to automate:
+
+- You want the continuous aggregate to be up to date with the
+  hypertable, even when you remove data from the underlying
+  hypertable.
+- You want to aggregate date from the hypertable into the continuous
+  aggregate and also want to keep aggregated data when removing data
+  from the underlying hypertable.
+
+Continuous aggregate policies are added with the function
+[`add_continuous_aggregate_policy`][add-continuous-aggregate-policy],
+which takes three arguments:
+
+- The parameter `start_interval` indicate the start of the refresh
+  window relative to the current time when the policy executes.
+- The parameter `end_interval` indicate the end of the refresh window
+  relative to the current time when the policy executes.
+- The parameter `schedule_interval` indicate the refresh interval in
+  wall-clock time.
+
+Similar to the `refresh_continuous_aggregate` function, providing
+`NULL` to `start_interval` or `end_interval` makes the range
+open-ended and will extend to the beginning or end of time,
+respectively.
+
+For example, to create a policy for `conditions_summary_hourly` that
+keeps the continuous aggregate up to date with the underlying
+hypertable `conditions` and run every hour, you would write:
+
+```sql
+SELECT add_continuous_aggregate_policy('conditions_summary_hourly',
+	start_interval => NULL,
+	end_interval => INTERVAL '1 h',
+	schedule_interval => INTERVAL '1 h');
+```
+
+This will ensure that all data in the continuous aggregate is up to
+date with the hypertable except the last hour and also ensure that we
+do not try to refresh the last bucket of the continuous
+aggregate. Since we give an open-ended `start_interval`, any data that
+is removed from `conditions` will also be removed from
+`conditions_summary_hourly`. In effect, the continuous aggregate will
+always reflect the data in the underlying hypertable.
+
+If you instead want to make sure that the aggregate data is kept in
+the continuous aggregate even if it is removed from the underlying
+hypertable, you can set a range for the `start_interval`. For example,
+if you have a [data retention policy][sec-data-retention] that removed
+data older than one month, you can set `start_interval` to one month
+(or less) and thereby not refresh data older than one month, which
+includes data that is removed.
+
+```sql
+SELECT add_continuous_aggregate_policy('conditions_summary_hourly',
+	start_interval => INTERVAL '1 month',
+	end_interval => INTERVAL '1 h',
+	schedule_interval => INTERVAL '1 h');
+```
+
```

Comment:
Maybe add something:

"It is important to consider the implications of continuous aggregate policies and data retention policies together, to ensure that dropping data from the underlying hypertable doesn't lead to overwriting aggregates in the continuous aggregate view with NULL values (unless desired).  For example, in this above example, if the data retention policy was instead  2 weeks while the continuous aggregate policy remained 1 month, after data more than 2 weeks is dropped, the next time the continuous aggregate is refreshed, the aggregate data from 2-4 weeks ago will be effectively deleted from the view."

Or something like that?

Reply:
I would not say that we overwrite with NULL values since that implies we actually write values as opposed to deleting.

Reply:
Added a warning about this, along the lines of what @mike suggested.